index.bs

```diff
@@ -1556,6 +1567,7 @@ input parameters:
     these are known to the authenticator, it should not create a new credential.
 - Extension data created by the client based on the extensions requested by the [=[RP]=], if any.
 - The |requireResidentKey| parameter of the |options|.{{MakePublicKeyCredentialOptions/authenticatorSelection}} dictionary.
+- The |requireUserVerification| parameter of the |options|.{{MakePublicKeyCredentialOptions/authenticatorSelection}} dictionary.
```

Comment:
please fix the ordering of the params as listed here and as listed in #createCredential alg, per comment here: https://github.com/w3c/webauthn/pull/502#pullrequestreview-46415425  -- i.e., i suggest keeping extension data as the last item in the params list, please see comment above.